src/json-schema/partial/map_state.json

```diff
@@ -49,6 +49,219 @@
+                                    "description": "A string that specifies the Amazon S3 API action Step Functions must invoke depending on the specified dataset.",
+                                    "examples": [
+                                        "arn:aws:states:::s3:putObject"
+                                    ]
+                                },
+                                "Parameters": {
+                                    "type": "object",
+                                    "description": "A JSON object that specifies the Amazon S3 resource to export the results to.",
+                                    "properties": {
+                                        "Bucket": {
+                                            "type": "string",
+                                            "description": "A string that specifies the Amazon S3 bucket name to export the results to.",
+                                            "minLength": 1
+                                        },
+                                        "Prefix": {
+                                            "type": "string",
+                                            "description": "A string that specifies the Amazon S3 object prefix to export the results to.",
+                                            "minLength": 1
+                                        }
+                                    },
+                                    "required": [
+                                        "Bucket", 
+                                        "Prefix"
+                                    ]
+                                }
+                            },
+                            "required": [
+                                "Resource", 
+                                "Parameters"
+                            ]
+                        },
+                        "ItemProcessor": {
+                            "type": "object",
+                            "description": "A JSON object that defines a state machine which will process each item or batch of items of the array.",
+                            "properties": {
+                                "ProcessorConfig": {
+                                    "type": "object",
+                                    "description": "A JSON object that specifies the Map state processing mode and definition.",
+                                    "properties": {
+                                        "Mode": {
+                                            "type": "string",
+                                            "description": "The Map state processing mode.",
+                                            "minLength": 1,
+                                            "enum": [
+                                                "DISTRIBUTED",
+                                                "INLINE"
+                                            ],
+                                            "examples": [
```

Comment:
Do we need the examples property if we're already using an enum? (repeated)

Reply:
I thought we did, but we don't actually.